recipes/lie-group-controllers/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} -GNinja .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_TESTING:BOOL=ON \
+      -DROBOMETRY_USES_SYSTEM_nlohmann_json:BOOL=ON
+
+cmake --build . --config Release
+cmake --build . --config Release --target install
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
+  ctest --output-on-failure -C Release
```

Comment:
The CI says `No tests were found!!!`. Is this intentional?

Reply:
See reply to https://github.com/conda-forge/staged-recipes/pull/20429/files#r982624520 .